packages/ag-charts-enterprise/src/components/dialog/dialog.css

```
@@ -146,6 +146,19 @@
     width: 26px;
 }
 
+.ag-charts-dialog__color-picker-button--multi-color {
+    background: linear-gradient(
+        135deg,
```

I think the designs wanted

```suggestion
        to right,
```

---

I think it was the one in annotationOptionsToolbar.css